ui/bookmark_groups/OBABookmarkGroupsViewController.m

```diff
@@ -58,24 +58,53 @@ - (void)close {
     [self dismissViewControllerAnimated:YES completion:nil];
 }
 
-- (void)addGroup {
-    UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Add Bookmark Group", @"") message:nil preferredStyle:UIAlertControllerStyleAlert];
+static UIAlertAction *saveButton;
```

Comment:
Why not make this a property on the object? If you want to keep this as is, please set to nil initially.